src/router/logged-in/screens/HomeScreen.tsx

```diff
@@ -91,6 +44,7 @@ const HomeScreen = ({ navigation }) => {
   const { logout } = useAuth();
   const { fetchUser, clearUserData } = useContext(UserContext);
   const { createAlert } = useAlert();
+  const links = t('links', { returnObjects: true }) as any;
```

Comment:
Can we make this more typesafe?

```suggestion
  const links = t('links', { returnObjects: true }) as Links;
```

With elsewhere:
```
interface Links {
  primary?: string[];
  secondary?: string[];
}
```